server/core_authenticate.go

```diff
@@ -469,6 +472,18 @@ func AuthenticateGoogle(ctx context.Context, logger *zap.Logger, db *sql.DB, cli
 		}
 	}
 
+	displayName := googleProfile.Name
+	if len(displayName) > 255 {
+		// Trim the name in case it is longer than db can store
+		displayName = displayName[0:255]		
```

Comment:
I think we should let displayName to be "" if we can't fully update it. We should also leave a WARN log to tell developers that this has happened.